### ISSUE TITLE
Chronic Lymphocytic Leukemia (CLL)/Small Lymphocytic Lymphoma (SLL)

### ISSUE THREAD

**Issue by galyea123** [OD4RD/Main-Help-Desk#182]
Hi!  As per the the fourth edition of the World Health Organization Classification of Tumours of Haematopoietic and Lymphoid Tissue, as well as the National Cancer Institute (NCI), both CLL and SLL are currently considered different manifestations of the same entity. 
https://www.nature.com/articles/s41375-022-01620-2/tables/1
https://lymphoma.org/wp-content/uploads/2022/07/LRF_CLL_SLL_Booklet_2022.pdf
https://pubmed.ncbi.nlm.nih.gov/23006943/
https://www.cancer.gov/publications/dictionaries/cancer-terms/def/chronic-lymphocytic-leukemia-small-lymphocytic-lymphoma 

However, I see that you do not list **chronic lymphocytic leukemia** in Orphanet (which in fact is not a rare disorder in Europe or in USA) but you do list **small lymphocytic lymphoma** in Orphanet (you  include SLL as a synonym for "B-cell chronic lymphocytic leukemia").

- In addition, according to the website Cancer.Net (of the American Society of Clinical Oncology) there are 2 general types of CLL based on whether the disease affects B cells or T cells (more than 95% of people with CLL have the B-cell type: B-cell chronic lymphocytic leukemia, and, about 1% of people with B-cell leukemia have a type called B-cell prolymphocytic leukemia (PLL).

-  The T-cell type of CLL is now called T-cell prolymphocytic leukemia. About 1% of people with CLL have the T-cell type. 

If this is the case,  B-cell chronic lymphocytic leukemia should be considered as a subtype of chronic lymphocytic leukemia.

Please review and clarify.

Thank you! Gioconda

Of note:  I plan on also contacting the NCI for further clarification.

**Comment by Orphanet-MartaFructuoso** [comment]
Dear @galyea123,

Thank you very much for your message and the references you provided.
Kindly update me as soon as you receive the input from the NHI.
Meanwhile, I will document and discuss the issue in our upcoming Orphanet medical and scientific committee meetings. 
After that, I will get back to you.

Thank you again for your help.
Best regards,
 
Marta

**Comment by galyea123** [comment]
Hi Marta!  Thank you very much for your response.  I received this message as well, from Dr. German Ott:

"chronic lymphocytic lymphoma is a term reserved for a B-cell lymphoma, and the other term is T-prolymphocytic leukemia which is exclusively a T-cell neoplasm.
Hence, CLL is always of B lineage. In addition, Daphne (de Jong) clarified that t seems that the Cancer.net site is quite haphazardly updated. There is rather relevant molecular information included, but the statements that are copied below are severely outdated. Gioconda might contact them to raw their attention to this outdated information."
Hope that helps!
Best regards
German

Professor Dr. German Ott
Institut für Klinische Pathologie
Robert-Bosch-Krankenhaus
Auerbachstrasse 110
70376 Stuttgart
Germany
Phone: +49 711 8101 3390" [end of message]

My next step is to contact the NCI and then I will get back to you.

Thank you and have a great weekend,

Gioconda

**Comment by Orphanet-MartaFructuoso** [comment]
Dear Gioconda,

Apologize for the delayed response.

Please, find here the information related to these entities we have currently in the Orphanet database:

- B-cell chronic lymphocytic leukemia, ORPHA:67038 [Disease] bellowing to the Group “Indolent B-cell non-Hodgkin lymphoma, ORPHA:300842”
The entity codes for (B-cell) chronic lymphoid leukemia (CLL) and Small lymphocytic lymphoma (SLL), terms appearing as synonyms.

- B-cell prolymphocytic leukemia, ORPHA:86852 [Disease] bellowing to the group “Aggressive B-cell non-Hodgkin lymphoma, ORPHA:300846”.
- 
-T-cell prolymphocytic leukemia, ORPHA:86871 [Disease] bellowing to the group “T-cell non-Hodgkin lymphoma, ORPHA:171918”, having “T-cell chronic lymphocytic” as a synonym. 

If I correctly understood your points, the codes cover the complete picture and consider the current terminology. Please, in case I did not reach the purpose of the request, can you rephrase or clarify what has to be revised or improved? 

Thank you in advance for your help!

Best regards,
Marta

**Comment by galyea123** [comment]
Hi Marta!  Thank you for your message.  My main question is if chronic lymphocytic leukemia (CLL) and B-cell chronic lymphocytic leukemia are in fact the same entity or different disorders.   As, according to the NCI https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&code=C3163 and other websites, as well as the opinion of Dr. Dr. German Ott (mentioned above) CLL is always of B lineage. 

If they are in fact synonyms, the term "chronic lymphocytic leukemia (CLL)" should be added to the term "B-cell chronic lymphocytic leukemia".  Thank you!  Gioconda (PS:  I am still waiting for the NCI response too).

**Comment by Orphanet-MartaFructuoso** [comment]
Dear Gioconda,
 
thank you for your message and clarifications. 
Best regards, 

Marta

**Comment by galyea123** [comment]
Thank you Marta! Hope you have a great weekend!

**Comment by Orphanet-MartaFructuoso** [comment]
Dear Gioconda,
Thank you for your help.
The requested synonyms have been added to ORPHA:67038.
Hope you have a great weekend too ;-)
Marta

**Comment by galyea123** [comment]
Thank you very much!  Have a great weekend you too!
